src/main/java/net/mcreator/ui/init/BackgroundLoader.java

```diff
@@ -45,8 +45,8 @@ public static List<Image> loadUserBackgrounds() {
 				try {
 					images.add(ImageIO.read(f));
 				} catch (IOException e) {
-					LOG.error("Can not load " + f.getName(), e.getMessage());
-					e.printStackTrace();
+					LOG.error("Can not load " + f.getName());

```

Comment:
e.getMessage should be replaced with first log print:

`LOG.error("Can not load " + f.getName(), e);`